src/internal_modules/roc_pipeline/receiver_slot.cpp

```diff
@@ -84,13 +84,13 @@ status::StatusCode ReceiverSlot::refresh(core::nanoseconds_t current_time,
     status::StatusCode code = status::NoStatus;
 
     if (source_endpoint_) {
-        if ((code = source_endpoint_->pull_packets(current_time)) != status::StatusOK) {
+        if ((code = source_endpoint_->pull_packets(0)) != status::StatusOK) {
             return code;
         }
     }
 
     if (repair_endpoint_) {
-        if ((code = repair_endpoint_->pull_packets(current_time)) != status::StatusOK) {
+        if ((code = repair_endpoint_->pull_packets(0)) != status::StatusOK) {
             return code;
         }
     }
```

Comment:
Since `current_time` is used only as a fallback when RTS isn't set, we can safely pass it to all calls I guess? Because why not to guarantee RTS for source and repair packets as well.